tools/repl.md

```diff
@@ -50,14 +50,14 @@ into the REPL. This is useful for importing some code you commonly use in the
 REPL, or modifying the runtime in some way:
 
 ```
-$ deno repl --eval 'import { assert } from "https://deno.land/std@$STD_VERSION/testing/asserts.ts"'
-Deno 1.14.3
+$ deno repl --eval --allow-net 'import { assert } from "https://deno.land/std@$STD_VERSION/testing/asserts.ts"'
```

Comment:
I get an error if I try to run this, it seems `--allow-net` should go before `--eval`

Reply:
nice catch